dev/intellij-setup.md

```diff
@@ -34,6 +34,14 @@ an alias name. You can do this in Using `File` -> `Project Structure...` -> `Pla
 ## Code Style
 The Code Style is available in XML format at [druid_intellij_formatting.xml](druid_intellij_formatting.xml) and can be [imported into IntelliJ](https://www.jetbrains.com/help/idea/2017.1/copying-code-style-settings.html).
 
+## Set Code Coverage Runner
+Druid CI checks are configured to enforce code coverage using JaCoCo. The checks will prevent PR from being merged 
+if test coverage of new added code is below the set threshold. You should run the tests locally to make sure that
+your code pass the coverage threshold. In IntelliJ, edit the template of JUnit tests in "Run Configurations..." 
+to use Jacoco for code coverage (this provides branch coverage instead of the default which is only line coverage).
+You can then right click src/test/java folder of the modules you are modifying and click run with coverage. This
+will generate a report to show the current code coverage on the code (not just your change).
+
```

Comment:
Can you add screenshots to this section to make it easier for devs to follow along

Reply:
Done